### PR TITLE
Add S3 Connector

### PR DESCRIPTION
**Pull request by ovekyc**
- Edit configuration
- Add AWS config json
- add S3Connector test code

### CODE REVIEW

**Comment by ochanje210** [comment]
LGTM

File: src/server/aws-s3.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import AWS from 'aws-sdk';
+import config from 'config';
+
+AWS.config.loadFromPath(config.awsConfig);
+
+export class S3Connector {
+  constructor() {
+    this.s3instance = new AWS.S3();
```
Comment (festinalente91) [inline src/server/aws-s3.js:8]: is there no connection limit in AWS?

Comment (ovekyc) [reply src/server/aws-s3.js:8]: IDK but i just made this class in simple way
I will edit this class to singleton mechanism if we need it

Comment (festinalente91) [reply src/server/aws-s3.js:8]: :+1:

**Comment by festinalente91** [comment]
LGTM

**Pull request merged**